.ci/on_conan_release.jenkinsfile

```diff
@@ -0,0 +1,87 @@
+/*
+
+    Triggered whenever a new Conan version is uploaded to pypi.
+
+*/
+
+node('Linux') {
+    stage('Input parameters') {
+        echo """
+        - conan_version: ${params.conan_version}
+        - build_new_images: ${params.build_new_images}
+        - build_old_images: ${params.build_old_images}
+        """
+    }
+
+    stage('New docker images') {
+        // Trigger the build (no need to wait)
+        if (params.build_new_images) {
+            build(job: 'ConanCenterIndex', wait: false)
+        }
+    }
+
+    stage('Old docker images') {
+        // Here we need to create a PR modifying the Conan version
+
+        // Clone, save and check if diff
+        checkout([$class           : 'GitSCM',
+                  branches         : [[name: '*/master']],
+                  userRemoteConfigs: [[credentialsId: 'conanci-gh-token', url: 'https://github.com/conan-io/conan-docker-tools.git']],
+                  extensions       : [[$class           : 'RelativeTargetDirectory',
+                                       relativeTargetDir: 'conan-docker-tools']],
+        ])
+
+        // Modify files that list Conan version. Change defaults to match this latest version
+        def content1 = readFile(file: 'conan-docker-tools/build.py')
+        content1 = content1.replaceAll(/TARGET_CONAN_VERSION = "\d+\.\d+.\d+"/, "TARGET_CONAN_VERSION = \"${params.conan_version}\"")
+        writeFile(file: 'conan-docker-tools/build.py', text: content1)
+
+        def content2 = readFile(file: 'conan-docker-tools/modern/.env')
+        content2 = content2.replaceAll(/CONAN_VERSION=\d+\.\d+.\d+/, "CONAN_VERSION=${params.conan_version}")
+        content2 = content2.replaceAll(/DOCKER_TAG=\d+\.\d+.\d+/, "DOCKER_TAG=${params.conan_version}")
+        writeFile(file: 'conan-docker-tools/modern/.env', text: content2)
+
+
+        String branchName = "bot/release-${params.conan_version}".replaceAll(/\./, '_')
+        dir('conan-docker-tools') {
```

Comment:
```suggestion
        // TODO: Check if this is already created, in case the stage `Open PR in Github` fails.
        dir('conan-docker-tools') {
```